src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.xml

```diff
@@ -17,11 +17,31 @@
         </service>
 
         <!-- Normalizer -->
-        <service id="serializer.normalizer.get_set_method" class="Symfony\Component\Serializer\Normalizer\GetSetMethodNormalizer" public="false">
+        <service id="serializer.normalizer.object" class="Symfony\Component\Serializer\Normalizer\ObjectNormalizer" public="false">
+            <argument type="service" id="serializer.mapping.class_metadata_factory" />
+
             <!-- Run after all custom serializers -->
             <tag name="serializer.normalizer" priority="-1000" />
         </service>
 
+        <!-- Loader -->
+        <service id="serializer.mapping.chain_loader" class="Symfony\Component\Serializer\Mapping\Loader\LoaderChain" public="false">
+            <argument type="collection" />
+        </service>
+
+        <!-- Class Metadata Factory -->
+        <service id="serializer.mapping.class_metadata_factory" class="Symfony\Component\Serializer\Mapping\Factory\ClassMetadataFactory" public="false">
+            <argument type="service" id="serializer.mapping.chain_loader" />
+            <argument>null</argument>
+        </service>
+
+        <!-- Cache -->
+        <service id="serializer.mapping.cache.apc" class="Doctrine\Common\Cache\ApcCache" public="false">
+            <call method="setNamespace">
+                <argument>%serializer.mapping.cache.prefix%</argument>
+            </call>
+        </service>
```

Comment:
For the record, though it still looks a little odd to me to have this service here that may not be used, I do like it. As I mentioned before, it's private, so will be removed if it's not being used. And I think it ends up being quite usable. :+1: